dags/graphics_telemetry.py

```diff
@@ -61,13 +62,15 @@
     else:
         aws_access_key, aws_secret_key, _ = AwsBaseHook(aws_conn_id=write_aws_conn_id, client_type='s3').get_credentials()
 
-    wait_for_main_ping = ExternalTaskCompletedSensor(
+    wait_for_main_ping = ExternalTaskSensor(
         task_id="wait_for_main_ping",
         external_dag_id="copy_deduplicate",
         external_task_id="copy_deduplicate_main_ping",
         execution_delta=datetime.timedelta(hours=2),
         check_existence=True,
         mode="reschedule",
+        allowed_states=['success'],
+        failed_states=['failed', 'upstream_failed'],
```

Comment:
`failed_states` are by default `None`, so they'll need to be listed here explicitly
https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/sensors/external_task/index.html#airflow.sensors.external_task.ExternalTaskSensor

Reply:
I'll look into using a global variable once we decide to migrate

Reply:
Sounds good.  At that point I'd also suggest either using a similar global variable for default allowed states or removing the explicit `allowed_states` parameter (since the default behavior for `allowed_states` is what we want anyway).